backend/src/main/java/com/htecgroup/skynest/model/entity/AccessType.java

```diff
@@ -0,0 +1,12 @@
+package com.htecgroup.skynest.model.entity;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public enum AccessType {
```

Comment:
Where do you use this enum?

Reply:
nowhere at the moment, but I'm sure whoever writes the permissions service will need this enum